tutorial/00_key_features/002_configurations.py

```diff
@@ -0,0 +1,106 @@
+"""
+.. _configurations:
+
+Pythonic Search Space
+=====================
+
+For hyperparameter sampling, Optuna provides the following features:
+
+- :func:`optuna.trial.Trial.suggest_categorical` for categorical parameters
+- :func:`optuna.trial.Trial.suggest_int` for integer parameters
+- :func:`optuna.trial.Trial.suggest_float` for floating point parameters
+
+With optional arguments of ``step`` and ``log``, we can discretize or take the logarithm of
+integer and floating point parameters.
+"""
+
+
+import optuna
+
+
+def objective(trial):
```

Comment:
In this file, the function `objective` does not return anything. It may confuse users. How about writing the entire function definition, or making it easy to understand that we skip latter part of this function?

Reply:
I think that caution is verbose rather than helpful. Call for a third opinion.